auth/store.go

```diff
@@ -587,6 +599,13 @@ func (as *authStore) AuthInfoFromToken(token string) (*AuthInfo, bool) {
 	as.simpleTokensMu.RLock()
 	defer as.simpleTokensMu.RUnlock()
 	t, ok := as.simpleTokens[token]
+	inf, timerok := as.simpleTokenInf[token]
+	if timerok {
+		if !inf.tokenTimer.Stop() {
+			<-inf.tokenTimer.C
+		}
+		inf.tokenTimer.Reset(SimpleTokenTTL)
```

Comment:
@xiang90 , here the timer is being reset if the client sends a request if the timer is not fired yet